tests/snapshots/tests.contrib.langchain.test_langchain_community.test_pinecone_vectorstore_asimilarity_search_with_score.json

```diff
@@ -0,0 +1,113 @@
+[[
```

Comment:
I think I'm not understanding what the different tagging we do (do we?) between similarity search with vector / by score that requires separate snapshot tests entirely. Can you correct me here? If not, could we try minimizing the number of snapshot tests (and file footprint) we add to the ddtrace project?